vmm/src/api/http/http_endpoint.rs

```diff
@@ -53,7 +52,7 @@ impl EndpointHandler for VmCreate {
                         }
 
                         match crate::api::VmCreate
-                            .send(api_notifier, api_sender, Arc::new(Mutex::new(vm_config)))
+                            .send(api_notifier, api_sender, Box::new(vm_config))
```

Comment:
It is possible to deserialize directly into the Box<VmConfig> that will remove the extra copy? My changing the type signature of `vm_config`

Reply:
Yes, it's possible! The suggestion also applies to `dbus`.